plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1287,6 +1293,7 @@ function VocabularyBuilderWidget:init()
         self.key_events.Close = { { Device.input.group.Back } }
         self.key_events.NextPage = { { Device.input.group.PgFwd } }
         self.key_events.PrevPage = { { Device.input.group.PgBack } }
+        self.key_events.ShowMenu = { { { "Menu" } }}
```

Comment:
Looks like a redundant pair of brackets.

Reply:
Fixed.